west.yml

```diff
@@ -43,8 +43,9 @@ manifest:
 
     - name: zephyr
       repo-path: zephyr
-      revision: e3ae110a05d3427647a03fcbeff4478c195c5a48
+      revision: pull/55667/head
```

Comment:
maybe keep this PR as draft until that Zephyr PR is merged? To make sure this isn't merged by accident